### PR TITLE
added .travis.yml

### PR DESCRIPTION
**Pull request by gururajbn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+
+install: python setup.py install
```
Comment (PrabhanshuAttri) [inline .travis.yml]: add script key with test command. [Reference](https://www.smartfile.com/blog/testing-python-with-travis-ci/)

**Comment by gururajbn** [comment]
@PrabhanshuAttri added `script` tag.

**Comment by PrabhanshuAttri** [comment]
@gururajbn did you test this?

**Comment by gururajbn** [comment]
@PrabhanshuAttri yes

**Comment by PrabhanshuAttri** [comment]
@gururajbn Closing this PR as @ashleyyy94 is working on it now. Fee free to contribute to this or any other repository you may find interesting. Will be making one more repository (Cloud based) public very soon. Thanks

**Pull request closed without merging**